opentelemetry-zio-logging/src/main/scala/zio/telemetry/opentelemetry/zio/logging/ZioLogging.scala

```diff
@@ -0,0 +1,60 @@
+package zio.telemetry.opentelemetry.zio.logging
+
+import io.opentelemetry.api.trace.{Span, SpanContext}
+import io.opentelemetry.context.Context
+import zio._
+import zio.logging.LogFormat
+import zio.logging.LogFormat.label
+import zio.telemetry.opentelemetry.context.ContextStorage
+
+trait ZioLogging {
+
+  /**
+   * Will print traceId from current span or nothing when not in span
+   */
+  def traceId(): LogFormat
```

Comment:
`()` are redundant => `def traceId: LogFormat`

Reply:
Same for the rest of the methods

Reply:
Fixed